roles/oneview_server_profile/tasks/main.yml

```diff
@@ -0,0 +1,121 @@
+---
+- name: Create a Server Profile from a Server Profile Template
+  oneview_server_profile:
+    config: "{{ config }}"
+    data:
+      serverProfileTemplateName: "{{ ov_template }}"
+      name: "{{ inventory_hostname }}"
+      # User should set initialScopeUris in case of scoped user
+      initialScopeUris:
+        - '/rest/scopes/12ab33bb-391f-491a-adfb-02b0dc625b3e'
```

Comment:
tasks file should be generic. Should not use hardcoded values.
Create a variable "scope_1" in defaults/main.yml file and use that variable name here as mentioned below
```
  - "{{ scope_1 }}"

Reply:
done